### ISSUE TITLE
Varying batch images size during training

### ISSUE THREAD

**Issue by tomad01** [eriklindernoren/PyTorch-YOLOv3#410]
Hi,
this is not an issue, it is more a curiosity. I may seem ignorant but I'm going to ask anyway:

I notices that for  every tenth batch you are using a new image size.
Is this some sort of augmentation ?
Can this be a problem considering that the anchors where calculated for 416 size ?

**Comment by Flova** [comment]
It is called multi-scale training and can be applied because the network is fully convolutional.
The anchor point is interesting, but I think it shouldn't affect it.

**Comment by LemonLov** [comment]
After resizing the image every time,  the ground truth bbox also needs to change. But it seems that you didn't change it?

**Comment by Flova** [comment]
The label notation is relative to the image size. This results in the labels being invariant against resolution changes.
They are converted to absolute labels at some points in the pipeline but this should not interfere with the resizing.
